### ISSUE TITLE
训练的loss不收敛

### ISSUE THREAD

**Issue by kingrt666666** [huawei-noah/Efficient-AI-Backbones#259]
老师您好！为什么使用VIG_S训练自己的数据的时候loss一直在震荡而没办法收敛呢。
![微信截图_20240426224743](https://github.com/huawei-noah/Efficient-AI-Backbones/assets/87821662/847af182-4ad8-41e5-84e0-b1d33da2d9a8)
下面是输出的summary.xls文件：<html xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/ADMINI~1/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/ADMINI~1/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#467886" vlink="#96607D">

epoch | train_loss | eval_loss | eval_top1 | eval_top5
-- | -- | -- | -- | --
  |   |   |   |  
0 | 1.467884 | 1.461614 | 22.4 | 100
  |   |   |   |  
1 | 1.395868 | 1.445047 | 23.475 | 100
  |   |   |   |  
2 | 1.380566 | 1.396237 | 26.65 | 100
  |   |   |   |  
3 | 1.38526 | 1.339708 | 27.55 | 100
  |   |   |   |  
4 | 1.388591 | 1.308261 | 26.55 | 100
  |   |   |   |  
5 | 1.397199 | 1.286827 | 28.075 | 100
  |   |   |   |  
6 | 1.398853 | 1.285091 | 35.15 | 100
  |   |   |   |  
7 | 1.403012 | 1.296711 | 43.675 | 100
  |   |   |   |  
8 | 1.407558 | 1.323002 | 46.9 | 100
  |   |   |   |  
9 | 1.410531 | 1.359494 | 29.75 | 100
  |   |   |   |  
10 | 1.405997 | 1.372851 | 25 | 100
  |   |   |   |  
11 | 1.406798 | 1.385787 | 25 | 100
  |   |   |   |  
12 | 1.402455 | 1.389221 | 25 | 100
  |   |   |   |  
13 | 1.4007 | 1.38869 | 25 | 100
  |   |   |   |  
14 | 1.397808 | 1.387725 | 21 | 100

</body>

</html>
训练的配置是：python train.py ./dataset/ --model vig_s_224_gelu --sched cosine --epochs 300 --opt adamw -j 8 --warmup-lr 1e-6 --mixup .8 --cutmix 1.0 --model-ema --model-ema-decay 0.99996 --aa rand-m9-mstd0.5-inc1 --color-jitter 0.4 --warmup-epochs 20 --opt-eps 1e-8 --repeated-aug --remode pixel --reprob 0.25 --amp --lr 2e-3 --weight-decay .05 --drop 0 --drop-path .1 -b 2 --output H:\\VIG-Pytorch\\vig_pytorch\\output
您能帮我看看吗？麻烦了

**Comment by iamhankai** [comment]
数据没有错的话，调调学习率
